android/src/main/kotlin/hk/gogovan/flutter_label_printer/FlutterLabelPrinterMethodHandler.kt

```diff
@@ -294,16 +333,271 @@ class FlutterLabelPrinterMethodHandler(
                         }
                     }
                 }
-                "hk.gogovan.label_printer.postfeedHMA300L" -> {
+                "hk.gogovan.label_printer.setPageWidthHMA300L" -> {
                     if (!PrinterHelper.IsOpened()) {
-                        result.error("1005", "Printer not connected.", Throwable().stackTraceToString())
+                        result.error(
+                            "1005",
```

Comment:
Better extract the error code and error msg to another place, to avoid further typos and misalignment. @luiges90

Reply:
These codes are shared between Dart layer and Native layer. Variables cannot be shared across this gap. I have already mitigated this by putting all error codes in `exception_codes.dart`.